### DISCUSSION TITLE
What should I do if I need a real or float?

### DISCUSSION THREAD

**Discussion by vkensou** [Whiley/WhileyCompiler#1162] [Q&A]
I know whiley has remove real. But what should I do if I really need a real or float?

**Comment by DavePearce** [comment]
Hey, 

That's a good question for which I don't have a good answer.  What I normally do in this case is use fixed-point arithmetic.  Roughly, you can have everything multiplied by e.g. `1000` to get 3 decimal places.  Some care is needed to make it work.
